src/main/java/com/dev/cinema/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -0,0 +1,51 @@
+package com.dev.cinema.service.impl;
+
+import com.dev.cinema.dao.ShoppingCartDao;
+import com.dev.cinema.dao.TicketDao;
+import com.dev.cinema.lib.Inject;
+import com.dev.cinema.lib.Service;
+import com.dev.cinema.model.MovieSession;
+import com.dev.cinema.model.ShoppingCart;
+import com.dev.cinema.model.Ticket;
+import com.dev.cinema.model.User;
+import com.dev.cinema.service.ShoppingCartService;
+import java.util.Collections;
+import java.util.List;
+
+@Service
+public class ShoppingCartServiceImpl implements ShoppingCartService {
+    @Inject
+    private ShoppingCartDao shoppingCartDao;
+
```

Comment:
do we need this separate?

Reply:
For me its more comfortable to have one more line between injectors :)